packages/forma-36-react-components/tools/.storybook/docs/General/General.stories.mdx

```diff
@@ -21,3 +21,43 @@ Design tokens can now be found on the [Forma 36 documentation website](https://f
 ## Components
 
 React components available for immediate use.
+
+### Library Usage
+
+Install the package to your project
+
+```bash
+yarn add @contentful/forma-36-tokens
+```
+
+Or for NPM
+
+```bash
+npm i @contentful/forma-36-tokens
+```
+
+Import into your project
```

Comment:
Is it clear enough that all of these are needed?

Reply:
hmm, yeah, maybe you are right. Maybe we could say: Import all the following into your project. Or something like that?

Reply:
w8 I think I made a mistake here. ups.